app/views/backend/fixed_assets/show.html.haml

```diff
@@ -1,3 +1,6 @@
+- if resource.started_on <= resource.journal.closed_on
+  .flash.warning
```

Comment:
Could you rather use the `notify_warning_now` method in the controller #show action rather than inserting it in the view directly?